.gitignore

```diff
@@ -18,3 +18,6 @@
 
 testdata
 *.png
+
+# Allow graphics used for bench test
+!ansi/fixtures/graphics/*.png
```

Comment:
I added `ansi/fixtures/graphics/JigokudaniMonkeyPark.png` originally but i have a concern if it would come along when someone would install x/ansi, i assume not right?